src/CoreWCF.NetTcp/tests/AuthorizationTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ServiceModel.Channels;
+using CoreWCF.Configuration;
+using CoreWCF.Description;
+using CoreWCF.IdentityModel.Policy;
+using CoreWCF.Primitives.Tests.CustomSecurity;
+using CoreWCF.Security;
+using Helpers;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CoreWCF.NetTcp.Tests
+{
+    public class AuthorizationTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public AuthorizationTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact(Skip ="Skipped in pipeline run")]
+        public void AuthorizationBasedonRolesTest()
```

Comment:
Formatting (need space after =), also can you update Skip comment to say needs to be running on domain joined machine.